### PR TITLE
Feature/AD-5 Devise User Setup

### PR DESCRIPTION
**Pull request by garethdandrews**
This PR covers 'AD-4 - add relevant gems to gemfile' and 'AD-5 - install devise and generate user model'.

The Gems for Devise, Bootstrap, Font Awesome and Faker have been added.
The user model has been created along with a seed to initialise the team as the first users.

### CODE REVIEW

**Review by AlexW555 — APPROVED** [review]
Looks all good to me! One thing to remember is to put @import in the application.scss in order to get bootstrap to initialise in the project. But I have already done that on my branch so no worries

File: Gemfile [hunk 1]
```diff
@@ -22,6 +22,11 @@ gem 'jbuilder', '~> 2.7'
 # Use Active Model has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+gem 'devise', '~> 4.8.0'
+gem 'bootstrap', '~> 5.0.1'
+gem 'font-awesome-sass', '~> 5.15.1'
+gem 'faker'
```
Comment (phil-6) [inline Gemfile:28]: Do you want faker in production too or just to be available in dev and test environments?

File: app/models/user.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```
Comment (phil-6) [inline app/models/user.rb:3]: do we also want confirmable?

File: Gemfile (hunk 1 above)
Comment (garethdandrews) [reply Gemfile:28]: Agreed. We might want to double check this in the Include platform.

Comment (garethdandrews) [reply Gemfile:28]: Depends if we want a demo on production, and if we want fake data on there?

File: app/models/user.rb (hunk 2 above)
Comment (garethdandrews) [reply app/models/user.rb:3]: and timeoutable and trackable?

**Comment by garethdandrews** [comment]
> Looks all good to me! One thing to remember is to put @import in the application.scss in order to get bootstrap to initialise in the project. But I have already done that on my branch so no worries

I was going to put that in a separate ticket but no worries.

File: Gemfile (hunk 1 above)
Comment (phil-6) [reply Gemfile]: I would say we probably only one one or two examples in production, which we could probably create as users. Good for testing that way too.

File: app/models/user.rb (hunk 2 above)
Comment (phil-6) [reply app/models/user.rb]: Timeoutable, maybe not, Trackable can be handy

**Review by phil-6 — APPROVED** [review]
Looks good

**Pull request merged**